src/Altinn.Platform/Altinn.Platform.Profile/Profile/Dockerfile

```diff
@@ -8,8 +7,7 @@ WORKDIR Profile/
 RUN dotnet build Altinn.Platform.Profile.csproj -c Release -o /app_output
 RUN dotnet publish Altinn.Platform.Profile.csproj -c Release -o /app_output
 
-#microsoft/dotnet:2.2-aspnetcore-runtime
-FROM microsoft/dotnet@sha256:7a9dfa52e5c02d1764964bbb034d9467d798020aa3747e2ddaea1ee3d2d386b8 AS final
+FROM mcr.microsoft.com/dotnet/core/runtime:3.0-alpine AS final
```

Comment:
Shouldn’t this instead be aspnet:3.0-alpine instead?
This is noe runtime sans asp.net, ref. https://hub.docker.com/_/microsoft-dotnet-core/

Reply:
I fix.